app/src/main/res/values-v27/styles.xml

```diff
@@ -20,7 +20,7 @@
     <style name="TuskyBlackTheme" parent="TuskyBlackThemeBase">
         <item name="android:windowLightNavigationBar">false</item>
         <item name="android:navigationBarColor">@color/black</item>
-        <item name="android:navigationBarDividerColor">@color/tusky_grey_05</item>
+        <item name="android:navigationBarDividerColor">@color/tusky_grey_10</item>
```

Comment:
I'm confused why it's different than splash above and why it's not attr

Reply:
Because I was not careful enough. Thx for checking!